From 9e509f4636d068367cb2cad21ee3c66af4efae50 Mon Sep 17 00:00:00 2001
From: Adeel Arshad <embeddi.zone@gmail.com>
Date: Sun, 5 Aug 2018 11:15:15 +0100
Subject: [PATCH] set fdt_file name

Signed-off-by: Adeel Arshad <embeddi.zone@gmail.com>
---
 include/configs/mx6var_som.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mx6var_som.h b/include/configs/mx6var_som.h
index 6d54c5c..d9b78b0 100644
--- a/include/configs/mx6var_som.h
+++ b/include/configs/mx6var_som.h
@@ -256,7 +256,7 @@
 	BOOT_ENV_SETTINGS \
 	VIDEO_ENV_SETTINGS \
 	OPT_ENV_SETTINGS \
-	"fdt_file=undefined\0" \
+	"fdt_file=imx6q-elecex-zgateway.dtb\0" \
 	"fdt_addr=0x18000000\0" \
 	"fdt_high=0xffffffff\0" \
 	"splashsourceauto=yes\0" \
-- 
2.7.4

